pyparsing/diagram/__init__.py

```diff
@@ -341,9 +354,9 @@ def _to_diagram_element(
             ret = EditablePartial.from_call(railroad.Sequence, items=[])
     elif isinstance(element, (pyparsing.Or, pyparsing.MatchFirst)):
         if _should_vertical(vertical, len(exprs)):
-            ret = EditablePartial.from_call(railroad.HorizontalChoice, items=[])
```

Comment:
Good catch, I evidently had this wrong. Let me know if it actually looks better this way, though.

Reply:
Was folded into master already.